src/xtgeo/grid3d/grid_properties.py

```diff
@@ -437,6 +435,8 @@ def from_file(
         else:
             raise OSError("Invalid file format")
 
+        self.append_props(properties)
+
     # def to_file(self, pfile, fformat="roff"):
```

Comment:
Any reason to keep commented code?

Reply:
Not really, I'll create a separate PR for it.